### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by rkzel**
Few small updates - fixed commands, added another cleaning command right before running start:with:profiles - styled-components come back after they are removed in first steps.

### CODE REVIEW

**Comment by Schwartz10** [comment]
Wondering why this changes 6 files when it's just changes to the readme. Also, please add back in the `<br />` to the readme that were taken out because the formatting looks all weird in the readme if you don't have those lol. Unless you know something that I don't!!

Instead of trying to do any fancy git maneuvers, you could simply:

1. Copy the entire readme.md file from this PR to your clipboard
1. `git checkout master`
2. `git pull origin master`
3. `git checkout -b update/readme`
4. paste the new readme
5. submit that PR

Then we won't have 6 files being changed in this commit.

**Comment by Schwartz10** [comment]
@rkzel when the readme update is in #29 go ahead and close this PR (don't merge)

**Review by Schwartz10 — CHANGES_REQUESTED** [review]
See comment above ^^

File: src/client/components/modals/Organization.js
Comment (chadoh) [inline src/client/components/modals/Organization.js:25]: I'll defer to `prettier` if we have it, but if not, it looks really strange to my eyes to see this line hanging out at this indentation level by itself, after the awkward-to-read `if` statement. Maybe we can rewrite this as:

```
if (!validateDAOAddress(
  getFormValue('organizations', organizationId, 'address')
)) {
  errors['organization'] = 'Please provide valid DAO address'
}

File: src/client/components/modals/Organization.js
Comment (chadoh) [inline src/client/components/modals/Organization.js:36]: I think a dash would make this more readable. "Enter an organization ID for which your logged-in ethereum account is a member" – @stellarmagnet what do you think?

File: src/client/utils/validation.js
Comment (chadoh) [inline src/client/utils/validation.js:16]: It looks like we're not currently checking that the logged-in ethereum id is a member of the provided organization. Is that accurate?

Screen Shot 2019-07-02 at 15 16 05

**Review by chadoh — APPROVED** [review]
Thanks for the README changes!

The other changes look good to me, but I'm not sure why they're part of the same PR as the README update. Do you mind splitting them into a separate PR, for better book-keeping?

File: README.md
Comment (chadoh) [inline README.md:36]: These are not showing up on separate lines in the rendered html.

Comment (chadoh) [reply README.md:36]: We can fix that by adding two spaces at the end of each line, or maybe better, putting them all in a code block. Indent each line four spaces, or wrap the block in triple backticks. Making this a bulleted or number list might also look nice.

File: README.md
Comment (chadoh) [inline README.md:46]: Same thing about line breaks here and throughout.

Also here: Should we add a line for which branch to checkout? "Get the development branch", but I don't think that branch is the default. Is `dev` the correct branch?

**Comment by rkzel** [comment]
Sorry for the delay and confusion - I've created a fresh, one-file-change, final PR: Updated readme file #58. Closing this one.

**Pull request closed without merging**
